pkgs/development/libraries/pipewire/default.nix

```diff
@@ -102,19 +102,6 @@ let
       ./0090-pipewire-config-template-paths.patch
       # Place SPA data files in lib output to avoid dependency cycles
       ./0095-spa-data-dir.patch
-
-      # FIXME: backports, remove after 0.3.64
-      # fix bluetooth issues
-      (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done.